### PR TITLE
Migration run each time while the service boot up

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java [hunk 1]
```diff
@@ -83,7 +84,10 @@ private static void migrateProjects() {
         Root<ProjectEntity> root = criteriaQuery.from(ProjectEntity.class);
 
         CriteriaQuery<ProjectEntity> selectQuery =
-            criteriaQuery.select(root).orderBy(criteriaBuilder.asc(root.get("id")));
+            criteriaQuery
+                .select(root)
+                .where(criteriaBuilder.equal(root.get("visibility_migration"), false))
+                .orderBy(criteriaBuilder.asc(root.get("date_created")));
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java]: We don't even need to order!

File: backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java [hunk 2]
```diff
@@ -83,7 +84,10 @@ private static void migrateProjects() {
         Root<ProjectEntity> root = criteriaQuery.from(ProjectEntity.class);
 
         CriteriaQuery<ProjectEntity> selectQuery =
-            criteriaQuery.select(root).orderBy(criteriaBuilder.asc(root.get("id")));
+            criteriaQuery
+                .select(root)
+                .where(criteriaBuilder.equal(root.get("visibility_migration"), false))
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java]: Should we add an index on this?

Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java]: Added index for the visibility_migration column

File: backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java]: Removed

**Review by conradoverta — APPROVED** [review]
LGTM. Excited to give this a go!

**Pull request merged**